src/pages/Art/Create/index.tsx

```diff
@@ -15,6 +15,7 @@ import {orderBy} from 'lodash-es';
 const Create: SFC = ({className}) => {
   const [createOpenAIImageResponse, setCreateOpenAIImageResponse] = useState<CreateOpenAIImageResponse | null>(null);
   const [description, setDescription] = useState('');
+  const [isImageSaved, setIsImageSaved] = useState<Array<number>>([]);
```

Comment:
`isImageSaved` sounds like a boolean but its a list of integers representing IDs/indexes of saved images, so I think we should name if accordingly

Something like
```suggestion
  const [savedImageIds, setSavedImageIds] = useState<Array<number>>([]);
```

OR 

```suggestion
  const [savedImageIndexes, setSavedImageIndexes] = useState<Array<number>>([]);
```

but I think it's not super critical so let's do it in a follow up PR